src/main/java/de/siphalor/modsoftheworld/client/MOTWClient.java

```diff
@@ -98,21 +104,19 @@ public static boolean loadLogo(Identifier logoId, String modName, SplashProvider
 	public static SplashProvider loadSplashes(ModMetadata modMetadata) {
 		if(modMetadata.containsCustomValue(SPLASHES_KEY.toString())) {
 			try {
-				Resource resource = MinecraftClient.getInstance().getResourceManager().getResource(Identifier.tryParse(modMetadata.getCustomValue(SPLASHES_KEY.toString()).getAsString()));
-				if(resource != null) {
```

Comment:
because this throws when null, this whole nested block moves out a tab, so you might want to review without whitespace changes.